lib/virt_feature_test_base.pm

```diff
@@ -48,16 +48,21 @@ sub run_test {
     die('Please override this subroutine in children modules to run desired tests.');
 }
 
+sub try_remove_commands_logs {
+    # ignore the attempt timing out with "timeout 20" which exits before the
+    # script_run internal timeout
+    unless (defined(script_run("rm -f /root/{commands_history,commands_failure}"))) {
```

Comment:
reconnect_when_ssh_console_broken is needed  when this script_run timeout

Reply:
Besides suggest to rename the function, eg reconnect_if_problematic,  because the main purpose of this function is actually to claim connection health via a simple script_run cmd, then if it is problematic, reconnect connection.

Reply:
Please see my updated version.